ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_losses.py

```diff
@@ -58,3 +58,114 @@ def test_log_poisson_loss(
         compute_full_loss=compute_full_loss,
         atol_=1e-2,
     )
+    
+#mse
+@handle_test(
+    fn_tree="functional.ivy.experimental.mse_loss",
+    dtype_and_true=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=1,
+        max_num_dims=3,
+    ),
+    dtype_and_pred=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=1,
+        max_num_dims=3,
+    ),
+    reduction=st.sampled_from(["none", "sum", "mean"]),
+)
+def test_mse_loss(
+    dtype_and_true,
+    dtype_and_pred,
+    reduction,
+    test_flags,
+    backend_fw,
+    fn_name,
+    on_device,
+    ground_truth_backend,
```

Comment:
The ground truth fixture has been hidden we no longer need to specify it here

Reply:
Sure @hirwa-nshuti. I'll update the function as per remark and run the unit tests. Shall update you when it's successful.